srcpkgs/psuinfo/template

```diff
@@ -0,0 +1,23 @@
+# Template file for 'psuinfo'
+pkgname=psuinfo
+version=1.0
+revision=1
+noarch=yes
+create_wrksrc=yes
```

Comment:
Should not be needed. You can remove this line and the `${pkgname}-${version}` parts after `vcopy` in `do_install`. `create_wrksrc` is only needed when you fetch multiple files or the fetched archive does have more then one single folder in it's root.

Reply:
Hopefully I applied all the changes you'd pointed out. Thank you very much for help.